### PR TITLE
Open Registration

### PR DESCRIPTION
**Pull request by RichiH**
Signed-off-by: Richard Hartmann <richih@richih.org>

### CODE REVIEW

File: content/2021-losangeles/registration.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+## Registration cost
+
+In-Person: $249 standard / $349 on-site (includes free virtual access)
+
+## Virtual-Only: $20
```
Comment (roidelapluie) [inline content/2021-losangeles/registration.md:5]: Should that be a title?

File: content/2021-losangeles/registration.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+[REGISTER NOW](https://www.cvent.com/events/kubecon-cloudnativecon-north-america-2020-virtual/archived-e4091d10480b45aeb2714b99ecc71858.aspx)
```
Comment (roidelapluie) [inline content/2021-losangeles/registration.md:7]:
```suggestion
<a class="btn btn-lg btn-default" href="https://www.cvent.com/events/kubecon-cloudnativecon-north-america-2020-virtual/archived-e4091d10480b45aeb2714b99ecc71858.aspx" target="_blank" role="button">
  <i class="fa fa-briefcase"></i>&nbsp;&nbsp;Register
</a>
```

**Comment by roidelapluie** [comment]
It looks like we have too many tabs...

toomanytabs

**Comment by RichiH** [comment]
Thanks, I hate it.
image

**Comment by RichiH** [comment]
@juliusv any quick-fix magic help?

**Comment by RichiH** [comment]
@roidelapluie I removed venue for now so we can get the news out. Not ideal, but works.

**Comment by juliusv** [comment]
This display probem still happens now if you make your browser window too narrow for all the tabs to show. So I guess it was already there before anyway.

I'm a bit busy at the moment, but if someone wants to have a look at it, https://stackoverflow.com/questions/43713419/my-navbar-is-displaying-too-many-items-and-overflows-how-can-i-collapse-it-at-a looks like a good first pointer.

**Comment by RichiH** [comment]
Aye, that's what I meant with my screenshot. Suggestion by Elna:

Overview tab 
- Overview 
- Sponsor
- Venue 
- Code of Conduct 

Diversity + Inclusion Tab  
- D+I info 
- Scholarship 

Registration Tab 
- Registration 
- Covid-19 

CFP Tab (after CFP closed, change this to Schedule tab) 

I am about to drop, but that should fix it the other way.

**Review by roidelapluie — APPROVED** [review]
Let's go for the temp fix?

**Pull request merged**